client/src/pages/HomePage.css

```diff
@@ -0,0 +1,130 @@
+@import "../index.css";
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.Homepage {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 20px;
+  background-image: url("../assets/images/Vector.png");
+  background-position: inherit;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 83dvh;
+}
+
+.Lettering {
+  position: absolute;
+  top: 200px;
+  left: 200px;
+  width: 250px;
+}
+
+.Where {
+  background-color: #53a4d7;
+  font-size: 18px;
+  color: white;
+}
+
+.femaleAbuSimbel {
```

Comment:
Please stay consisent with selector nomenclature. Either use PascalCase or kebab-case everywhere, but not a mix of both.